docs/General/buildRPC.md

```diff
@@ -35,19 +35,23 @@ Read more about GitHub Actions [here](https://github.blog/2019-08-08-github-acti
 
 3. Enable workflows by clicking **I understand my workflows, go ahead and enable them**.
 
-4. Type "v" to see the list of release branches from the **Use workflow from** dropdown.
+4. Choose a workflow.
```

Comment:
We should specifically call out which workflow.  The only one users will ever use is the following.

```suggestion
4. Choose the **Build RPC (Native) Debug/Release** workflow.
```